src/ui/common/content/ContentsContainer.js

```diff
@@ -69,31 +48,26 @@ export const mapStateToProps = (state) => {
     sortingColumns: getSortingColumns(state),
     selectedRows: getSelectedRows(state),
     currentUsername: getUsername(state),
+    ownerGroup,
   });
 };
 
-export const mapDispatchToProps = (dispatch, { status: contentStatus, author: contentAuthor }) => ({
+export const mapDispatchToProps = (dispatch, { ownerGroup, joinGroups }) => ({
   onDidMount: () => {
-    dispatch(fetchContentsPaged(paramsForStatusAndAuthor(contentStatus, contentAuthor),
-      null, null, false));
+    dispatch(checkStatus('published'));
+    dispatch(fetchContentsPaged(null,
+      null, null, false, '&status=published', ownerGroup, joinGroups));
     dispatch(fetchCategoryTree());
     dispatch(fetchGroups({ page: 1, pageSize: 0 }));
     dispatch(fetchContentTypeListPaged({ page: 1, pageSize: 0 }));
   },
   onSetQuickFilter: filter => dispatch(setQuickFilter(filter)),
-  onFilteredSearch: (fetchParams, pagination, sortParams, tabSearch) => dispatch(
-    fetchContentsPaged(fetchParams, pagination, sortParams, tabSearch),
+  onFilteredSearch: (fetchParams, pagination, sortParams) => dispatch(
+    fetchContentsPaged(null,
+      pagination, sortParams, false, '&status=published', ownerGroup, joinGroups),
   ),
-  onSetCurrentAuthorShow: (author, status) => {
-    dispatch(setCurrentAuthorShow(author));
-    dispatch(fetchContentsPaged(paramsForStatusAndAuthor(status, author)), null, null, true);
-  },
-  onSetCurrentStatusShow: (status, author) => {
-    dispatch(setCurrentStatusShow(status));
-    dispatch(fetchContentsPaged(paramsForStatusAndAuthor(status, author)), null, null, true);
-  },
   onAdvancedFilterSearch: () => {
-    dispatch(fetchContentsPaged(null, null, null, false));
+    dispatch(fetchContentsPaged(null, null, null, false, '&status=published', ownerGroup, joinGroups));
```

Comment:
it's better to provide an object as param, instead of having all those param that are not easy to understand (e.g. null, null, null, false)